react/features/base/devices/reducer.js

```diff
@@ -0,0 +1,62 @@
+import {
+    SET_AUDIO_INPUT_DEVICE,
+    SET_AUDIO_OUTPUT_DEVICE,
+    SET_VIDEO_INPUT_DEVICE,
+    UPDATE_DEVICE_LIST
+} from './actionTypes';
+
+import { ReducerRegistry } from '../redux';
+
+const defaultState = {
+    audioInput: [],
+    audioOutput: [],
+    videoInput: []
+};
+
+/**
+ * Listen for actions which changes the state of known and used devices.
+ *
+ * @param {Object[]} state - Current state.
```

Comment:
I think I meant an Object of arrays but realized I should be more clear and also change the description to match what's used elsewhere for reducer descriptions.